### DISCUSSION TITLE
[Kavita] ”Art Book“ and "Guide Book" Relationships

### DISCUSSION THREAD

**Discussion by newyorker123** [Kareadita/Kavita#3261] [Ideas]
### Idea Description

Art books and guide books are very common in manga; however, the current "Relationships" section does not have appropriate relationship types for these. Therefore, I would like to propose adding relationship types for art books and guide books.

### Idea Category

User Experience

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by majora2007** [comment]
Why not just have them under specials with the main series? Generally art books don't have their own metadata entry in metadata providers.

**Reply by newyorker123** [reply]
In my case, the artbook and the main volumes are from different publishers. When I set the artbooks as "Specials," the publisher of the artbook is shown on the series page, instead of the main volumes' publisher, which is quite frustrating for me.

Actually, that's another issue I'd like to report. It's great to have an info page for each volume and Specials in v0.8.3. However, I would like to have some control over which value is displayed on the series page when a field has different values across these volumes.

**Reply by majora2007** [reply]
Don't report issues here, they don't get worked on. It sounds like there is a bug. If you can create an issue. Publisher is derived from the first issue or volume, it should not come from special unless nothing else is available.
